tests/phpunit/includes/SettingsTest.php

```diff
@@ -99,6 +99,32 @@ public function testRegisterChangeListener() {
 		$instance->clearChangeListeners();
 	}
 
+	public function testOverride_ListingToOverride() {
+
+		$changeListener = $this->getMockBuilder( '\SMW\Listener\ChangeListener\ChangeListener' )
```

Comment:
using `ChangeListener::class` helps static analysis tools find usage of `ChangeListener`